.gitignore

```diff
@@ -30,3 +30,5 @@ yarn-debug.log*
 yarn-error.log*
 
 /src/contracts/models
+
+package-lock.json
```

Comment:
The `package-lock,json` should absolutely not be added to the .gitignore. Please see [this](https://stackoverflow.com/a/48524475) for more info.

Reply:
Interesting. Makes sense. will remove from `.gitignore`. This is why we code review :)